e2e-tests/cypress/e2e/journeys/insurance/check-your-answers/your-buyer/change-your-answers/change-your-answers-change-working-with-buyer.spec.js

```diff
@@ -0,0 +1,160 @@
+import { submitButton } from '../../../../../pages/shared';
+import partials from '../../../../../partials';
+import { ROUTES, FIELD_IDS } from '../../../../../../../constants';
+import { checkYourAnswersYourBuyer } from '../../../../../pages/insurance/check-your-answers';
+import { workingWithBuyerPage } from '../../../../../pages/insurance/your-buyer';
+import {
+  checkChangeLinkUrl,
+  checkChangeAnswerRendered,
+} from '../../../../../../support/check-summary-list-field-change';
+
+const {
+  ROOT,
+  CHECK_YOUR_ANSWERS: {
+    YOUR_BUYER,
+  },
+  YOUR_BUYER: {
+    WORKING_WITH_BUYER_CHECK_AND_CHANGE,
+  },
+} = ROUTES.INSURANCE;
+
+const {
+  INSURANCE: {
+    YOUR_BUYER: {
+      WORKING_WITH_BUYER: {
+        CONNECTED_WITH_BUYER,
+        TRADED_WITH_BUYER,
+      },
+    },
+  },
+} = FIELD_IDS;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.submitApplication.tasks.checkAnswersAndSubmit;
+
+const { summaryList } = checkYourAnswersYourBuyer;
+
+const getFieldVariables = (fieldId, referenceNumber) => ({
+  route: WORKING_WITH_BUYER_CHECK_AND_CHANGE,
+  checkYourAnswersRoute: YOUR_BUYER,
+  newValueInput: '',
+  fieldId,
+  referenceNumber,
+  summaryList,
+  changeLink: summaryList.field(fieldId).changeLink,
+});
+
+context('Insurance - Check your answers - Working with buyer - Your buyer - Summary list', () => {
+  let referenceNumber;
+  let url;
+
+  before(() => {
+    cy.completeSignInAndGoToApplication().then((refNumber) => {
+      referenceNumber = refNumber;
+
+      cy.completePrepareApplicationSinglePolicyType();
+
+      task.link().click();
+
+      // to get past eligibility check your answers page
+      submitButton().click();
+      // to get past policy and exports check your answers page
+      submitButton().click();
+      // to get past your business check your answers page
+      submitButton().click();
```

Comment:
Not a blocker - could do with a cypress command for this, will need this eventually

Reply:
will do this in a later PR